README.md

```diff
@@ -1,199 +1,201 @@
+- `opts.start` *String* - feed ID of the "central" node where distance is zero.
+(Default: `sbot.id`)
+- `opts.max` *Number* - a max distance, where nodes beyond this distance are
+omitted from the output. If the max is equal to or less than the default
+(`config.friends.hops`), the output will be faster to calculate, because it will
+just copy the cached value, but skip nodes at a greater distance than max.
+(Default: `config.friends.hops` or 3)
+- `opts.reverse` *Boolean* - when `true`, the output is the hops distance **to*
+`opts.start`, instead of **from** `opts.start`. (Default: `false`)
 
-calls back true if `source` blocks `dest`, false otherwise.
-
-### ssb.friends.createLayer(name)`
-
-Create a layer with `name`, this feature is exposed from [layered-graph](https://github.com/dominictarr/layered-graph)
-
-This enables plugins to provide different views on feed relationships, and then combine them together. This method is not available remotely (over RPC).
-
-As an example, here is code to make a view that only track old-style pub follows.
-
-``` js
-  var layer = sbot.friends.createLayer('pubs')
-  var init = false //keep track of wether we have initialized the layer.
-
-  var view = sbot._flumeUse('pubs', Reduce(1, function (g, data) {
-    g = g || {}
-    var content = data.value.content
-    if(
-      content.type === 'contact' &&
-      isFeed(content.contact) &&
-      content.following === true &&
-      (content.autofollow === true || content.pub === true)
-    ) {
-      var from = data.value.author, to = content.contact
-      g[from] = g[from] || {}
-      g[from][to] = 1
-      //updating an edge in the layer should be handled within the view reduce function,
-      //but only after it has been initialized.
-      if(init) layer(from, to, 1)
-    }
-    return g
-  })
-
-  //if we call view.get like this, it will delay until this view is in sync with the main log.
-  //
-  view.get(function (err, value) {
-    init = true
-    layer(value)
-  })
+### `ssb.friends.hopStream([opts]) => Source`
 
-```
+Return a stream of hops objects `{<id>:<dist>,...}`, where the first item is the
+current state (such as what `ssb.friends.hops()` returns), and any following
+objects are updates caused by someone in your network following, unfollowing or
+blocking someone.
 
-## dynamic dijkstra
+Can be configured via an `opts` argument, although arguably *less* configurable
+than `ssb.friends.hops()` because it only supports the following fields:
 
-Since version 3, this module is now implemented in terms of [dynamic-dijkstra](https://github.com/dominictarr/dynamic-dijkstra) (via [layered-graph](https://github.com/ssbc/layered-graph)).
-DD is about traversing graphs that have real time updates.
+- `opts.old` *Boolean* - whether or not to include the current state (such as
+what `ssb.friends.hops()` returns). (Default: `true`)
+- `opts.live` *Boolean* - whether or not to include subsequent updates.
+(Default: `false`)
```

Comment:
I had the same thoughts. Turns out, `hopStream` always had defaults `live: false` and `old: true` because of how `layered-graph` is built: https://github.com/ssbc/layered-graph/blob/e29ff59b8a3f6e3aaabb2ebb86510f47f7958f5b/index.js#L129-L130

So I decided to keep `graphStream` analogous to that.

But I agree with you that it doesn't make much sense, and I'm willing to change the defaults since we're anyway doing a breaking change.

Reply:
Done!